.vscode/settings.json

```diff
@@ -0,0 +1,32 @@
+{
+    "python.testing.pytestArgs": [
+        "."
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.pythonPath": "C:\\Users\\Sultan\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\todo-app-4_7iQs12-py3.9\\Scripts\\python.exe",
```

Comment:
You probably want to gitignore this file.

Reply:
Thank you